internal/tests/integration/dataframe_test.go

```diff
@@ -694,3 +696,31 @@ func TestDataFrame_FreqItems(t *testing.T) {
 	assert.NoErrorf(t, err, "%+v", err)
 	assert.Len(t, res, 1)
 }
+
+func TestDataFrame_WithOption(t *testing.T) {
+	ctx, spark := connect()
+	filepath := "/tmp/test.csv"
+	file, err := os.Create(filepath)
+	assert.NoError(t, err)
```

Comment:
Can you please use a proper tempfile:

```golang
f, err := os.CreateTemp("", "example")
assert.NoError(t, err)
defer os.Remove(f.Name()) // clean up
```

Reply:
thanks. I didn't think of that and i have changed it now.